plugins/network-policy-audit/tests/validation/basic_functionality_test.sh

```
@@ -0,0 +1,211 @@
+echo "Test ID: ${TEST_ID}"
+echo ""
+
+set +e  # Don't exit on errors - we want to test error handling
+
+# Start test log
+{
+    echo "=== BASIC FUNCTIONALITY TESTS ==="
+    echo "Date: $(date)"
+    echo ""
+
+    cd ${PLUGIN_DIR}
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)